src/libelf/elf_update.c

```diff
@@ -502,7 +502,7 @@ _libelf_resync_sections(Elf *e, off_t rc, struct _Elf_Extent_List *extents)
 static off_t
 _libelf_resync_elf(Elf *e, struct _Elf_Extent_List *extents)
 {
-	int ec, eh_class, eh_type;
+	int ec, eh_class, eh_type __attribute__((unused));
```

Comment:
I prefer the `(void)x` style of marking things unused as it's compiler agnostic, but realistically this isn't going to get compiled by anything other than a gcc compatible compiler, so this attribute is fine.